src/features/common/Layout/Header/AccountHeader.module.scss

```diff
@@ -0,0 +1,90 @@
+@import './../../../../theme/theme';
+
+.headerBG {
+  height: 320px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: linear-gradient(
+      to top,
+      rgba(0, 0, 0, 0.5),
+      rgba(0, 0, 0, 0.2),
+      rgba(0, 0, 0, 0),
+      rgba(0, 0, 0, 0)
+    ),
+    url('https://cdn.plant-for-the-planet.org/media/images/app/bg_layer.jpg')
```

Comment:
I still want to ask if it's possible to set this image within the JavaScript code to allow us to use the env variable CDN_URL for the URL in order to be able to support different environments.

Reply:
Yes, we could do it but the CDN for development doesn't have this image 😅

Reply:
@sagararyal can we deploy above image to all CDN (develop, staging, production)?